### DISCUSSION TITLE
Need help deciphering non descriptive error code 5003

### DISCUSSION THREAD

**Discussion by HVSATSC3** [Haivision/srt#3042] [Q&A] [answered]
Hello I am trying to do an SRT connection (call out) and every time I try I get an error of #5003.0 Operation not supported: Bad Parameters error SRT_connect: operation not supported bad parameters. Even if I remove the extra parameters of passphrase adapter and mode it still fails. the listening end seems confident in their setup but I have never run into something like this before. 
![SRT issue](https://github.com/user-attachments/assets/122d1dc2-1721-4f87-b6e0-1e0f9394ea33)
Is the issue them using a domain instead of a public IP address? My computer is connected to a working DNS.

**Comment by ethouris** [comment]
Add `-ll debug` to the command line and show the result. You might need to enable heavy logging explicitly to see all possible debug logs, but try without first.

**Reply by HVSATSC3** [reply]
Adding the debug command nets a few more lines of output:
![SRT issue Debug](https://github.com/user-attachments/assets/32d9ceba-3c41-40a7-85b3-a8d18b6dfbca)

**Reply by maxsharabayko** [reply, marked as answer]
I would advise checking if your `*.com` is resolved properly.

**Reply by HVSATSC3** [reply]
I am checking in with the lab on the other end and having them check. I will update when I know more.

**Reply by HVSATSC3** [reply]
They gave me the domain out of order. I am connected up now thanks!
